core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -90,11 +141,11 @@ oppia.controller('ExplorationHistory', [
             'commitMessage': explorationSnapshots[i].commit_message,
             'versionNumber': explorationSnapshots[i].version_number
           };
-          $scope.snapshotOrderArray.push(explorationSnapshots[i].version_number);
+          $scope.snapshotOrderArray.push({vnum:explorationSnapshots[i].version_number, selected:false});
```

Comment:
I would suggest using versionNumber instead of vnum, and renaming snapshotOrderArray to versionCheckboxArray (since it no longer just stores the order of displayed snapshots).